rules/rule-identify-blocklisted-image-registries/raw.rego

```diff
@@ -76,9 +76,4 @@ untrusted_or_public_registries(image){
 	public_registries := data.postureControlInputs.publicRegistries
 	repo_prefix := public_registries[_]
 	startswith(image, repo_prefix)
-}
-
-untrusted_or_public_registries(image){
```

Comment:
Can you please explain why removing this function solves the issue?

Reply:
this function fails every image which does not contain "/". Not sure why it was there in the 1st place.. So in his case that's why it fails, and when changing the image to "docker.io/nginx:1.14.2" it passes, since there is a slash.